docs/Administrator Guide/Setting Up Volumes.md

```diff
@@ -581,6 +409,172 @@ volumes, but using dispersed subvolumes instead of replicated ones.
 
     > Use the `force` option at the end of command if you want to create the volume in this case.
 
+## Creating Striped Volumes
```

Comment:
do we need this as with latest version (glusterfs-6.x and glusterfs-7.x), these commands doesn't even work. I am OK with removing them altogether.

Reply:
Oh, I didn't realise they no longer worked, only just started using GlusterFS. Will push another commit to remove them. Is it all the deprecated volumes that have been removed or just the Striped ones?

Reply:
> only just started using GlusterFS.

Welcome! and much appreciated contribution Indeed.

We can remove the 'Striped' ones in this PR, and pick others in follow up patches as and when you encounter differences.

Reply:
> Welcome! and much appreciated contribution Indeed.

Thanks for the kind words and for your contributions as well! 😁

> We can remove the 'Striped' ones in this PR, and pick others in follow up patches as and when you encounter differences.

Realised when I looked again all the deprecated ones are 'Striped'! I've pushed another commit removing all the deprecated 'Striped' sections.

Reply:
We normally 'squash and merge' patches. So no need for that.